app/Domain/Dashboard/Templates/show.blade.php

```diff
@@ -94,11 +94,11 @@ class="{{ $isFavorite ? 'btn-active' : '' }}">
                 </x-global::content.card>
 
                 <x-global::content.card variation="content">
-                    <x-slot:card-context-buttons>
-                        <x-global::forms.button href="#/tickets/newTicket" content-role="tertiary">
+                    {{-- <x-slot:card-context-buttons>
```

Comment:
Please take out code that is not needed anhmore

Reply:
done